src/oidc/oidc.ts

```diff
@@ -53,7 +53,7 @@ export const fetchOidcConfiguration = async (): Promise<OidcConfiguration> => {
         const config = JSON.parse(configFromLocalStorage) as OidcConfiguration;
         // we need to check this, because in case when they go to the endpoints page and switch the app ID and server URL
         // we need to invalidate our OIDC configurations cuz its still pointing to the old server URL
-        if (config.issuer === serverUrl) return JSON.parse(configFromLocalStorage);
```

Comment:
` const { serverUrl } = getServerInfo();` returns `oauth.deriv.com` by default, which won't match production's `https://oauth.deriv.com` issuer, thus the condition `oauth.deriv.com === https://oauth.deriv.com` will always be false and the local storage cache won't be used